ci-operator/config/openshift/cloud-provider-alibaba-cloud/openshift-cloud-provider-alibaba-cloud-master.yaml

```diff
@@ -44,6 +44,16 @@ tests:
   commands: make gofmt
   container:
     from: src
+- as: e2e-alibabacloud-ccm
```

Comment:
Why `-ccm` variant? That implies a custom feature gate? For alibaba CCM is always applied so we don't need the feature gate